app.py

```diff
@@ -271,56 +271,49 @@ def convertPdfToText(event, context):
                 raise
 
     # Start converting PDF into JPEG Images
-    page_num = 0
+    n_pages = 0
     result = ''
     if os.path.isfile(pdf_file):
         app.log.info("Converting PDF to images!")        
-        pdf = pdfium.PdfDocument(pdf_file)
+        pdf = pdfium.PdfDocument(pdf_file, may_init_forms=True)
         version = pdf.get_version()  # get the PDF standard version
-        page_num = len(pdf)  # get the number of pages in the document
-        app.log.info("PDF has version " + str(version) + " and " + str(page_num) + "pages")
-        y = 0
-        #workaround since pdf.render_to uses parallel processing with queue which is not supported by Lambda
```

Comment:
I suppose you overlooked the single-page rendering method? Your code was quite some API misuse, slightly hurts to see as lib dev ;)